src/datadog_directive.cpp

```diff
@@ -2,22 +2,74 @@
 
 #include <cassert>
 
+#include "datadog_conf_handler.h"
+
 namespace datadog {
 namespace nginx {
 
-char *warn_deprecated_command_datadog_tracing(ngx_conf_t *cf,
-                                              ngx_command_t * /*command*/,
-                                              void * /*conf*/) noexcept {
+char *silently_ignore_command(ngx_conf_t *cf, ngx_command_t *command, void *) {
+  ngx_conf_log_error(NGX_LOG_DEBUG, cf, 0, "Directive \"%V\" ignored",
+                     &command->name);
+  return NGX_OK;
+}
+
+char *alias_directive(ngx_conf_t *cf, ngx_command_t *command, void *) noexcept {
```

Comment:
I have a few comments on this function:
- Can't we replace the assertion by a return of NGX_CONF_ERROR ? Because at the moment it would mean that the program is just crashing, no ?
- Do we really want to log the alias every times ?

Reply:
Auto answer: if the DEBUG_LOG are something we need to activate you can ignore my second question

Reply:
> Can't we replace the assertion by a return of NGX_CONF_ERROR ?

That's an interesting take. I do believe assert should be enabled in release build but that's not the case at the moment. Assertion in general are very unlikely scenarios that you mean to catch during development.

Here, it is _very_ unlikely because the NGINX API guarantee that the number of element is X if the directive is correctly defined. If this invariant is false, either the directive definition is wrong and will be catch during testing OR NGINX do not behave as expected.

> Do we really want to log the alias every times ?
It's a DEBUG_LOG, by essence, debug logs are quite verbose. Consequently, this is a non-issue.